frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.tsx

```diff
@@ -66,13 +67,32 @@ export function PollingStationChoiceForm({ anotherEntry }: PollingStationChoiceF
     }
   };
 
+  const inProgress = electionStatus.statuses
+    .filter((status) => status.status === "first_entry_in_progress")
+    .map((status) => pollingStations.find((ps) => ps.id == status.id));
+
   return (
     <form
       onSubmit={(e) => {
         e.preventDefault();
         return;
       }}
     >
+      {inProgress.length > 0 && (
+        <div className={cls["in-progress-alert"]}>
+          <Alert type="notify" variant="no-icon">
+            <h2>Je hebt nog een openstaande invoer</h2>
+            <p>
+              Je bent begonnen met het invoeren van onderstaande stembureaus, maar hebt deze nog niet helemaal afgerond:
+            </p>
+            {inProgress.map((pollingStation) => {
+              return pollingStation === undefined ? null : (
+                <PollingStationLink key={pollingStation.id} pollingStation={pollingStation} />
+              );
+            })}
+          </Alert>
+        </div>
+      )}
```

Comment:
The design has an [x] button to close this alert that's still missing in the implementation.

Reply:
Yes, good catch, I forgot about that one. Fixed in [38c267a](https://github.com/kiesraad/abacus/pull/404/commits/38c267a03e0b3bf0b5759e4c75971685d391552e)